python/private/stage1_bootstrap_template.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -e
```

Comment:
What about undefined vars?
```suggestion
set -eu
```

Although I see that it would cause a few problems with how we use `-z` right now.

Reply:
It would seem -u causes empty arrays to be treated as undefined on some platforms. Probably depending on the bash version? My local bash is OK with it, but not the mac/RBE shells. Looking around, it seems properly handling empty arrays with -u is very baroque, so I removed -u. (the empty array in question is interpreter_args, but I'd like to land another pr to allow populating that, and then this would be an annoying problem).